07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        pd.concat(params.values(), axis='rows')
</pre><p>I usually use dicts exactly as you do to gather the data that will become a dataframe later, but instead of running <code>concat</code> I use the <code>pd.DataFrame</code> constructor. If the dict has <code>Series</code> (see previous suggestion), that works smoothly. Something to check if it works for you in the future, if you find it useful and more direct.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I'll keep this in mind for the future! I'm going to stick with what I have for now but manipulating <code>Series</code> instead is a good idea. </p>